README.md

```diff
@@ -114,16 +106,25 @@ markdown. For example:
     <!!-- 
     ```variables
     export MY_RESOURCE_GROUP_NAME=myResourceGroup22323
-    export MY_LOCATION=eastus
+    export REGIONN=eastus
     export MY_VM_NAME=myVM
     export MY_VM_IMAGE=debian
     export MY_ADMIN_USERNAME=azureuser
     ```
     -->
+3. Pass in variable declarations as an argument to the ie CLI command using the 'var' parameter. For example:
+   ```bash
+	ie execute tutorial.md --var REGION=eastus
+   ```
+
+CLI argument variables override environment variables declared within the markdown document,
+which override variables set in a .ini file, which override preexisting environment variables.
```

Comment:
I'm impartial either way, removed for now to keep things more concise.  Let's keep the functionality though assuming no effort is needed to maintain it.  But there is a request to expand on the ini doc to install dependencies in #152 .  Although this request would also be solved by pre-reqs.

Reply:
IMO the INI file wouldn't be able to effectively solve #152 because there's no good way to resolve package names across various Linux distributions. Each distribution uses a different package management tool and each the name of the packages can be different across distributions although containing the same content as each other. Pre-requisites is definitely the preferred solution for this problem.